python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -680,6 +682,81 @@ def transformers_int4_npu_win(repo_id,
+                                       local_model_hub,
+                                       in_out_pairs,
+                                       warm_up,
+                                       num_trials,
+                                       num_beams,
+                                       low_bit,
+                                       batch_size,
+                                       optimize_model,
+                                       transpose_value_cache,
+                                       npu_group_size):
+    from ipex_llm.transformers.npu_model import AutoModel, AutoModelForCausalLM
+    from transformers import AutoTokenizer, LlamaTokenizer
+
+    model_path = get_model_path(repo_id, local_model_hub)
+    in_out_len = in_out_pairs[0].split("-")
+    max_context_len = max(int(in_out_len[0]) + int(in_out_len[1]), 1024)
+    # Load model in 4 bit,
+    # which convert the relevant layers in the model into INT4 format
+    st = time.perf_counter()
+    if repo_id in CHATGLM_IDS:
+        model = AutoModel.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True,
+                                          optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]),
+                                          quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
+                                          torch_dtype=torch.float16, attn_implementation="eager").eval()
+        tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
+    elif repo_id in LLAMA_IDS:
+        model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
+                                                     optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
+                                                     quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
+                                                     use_cache=True, attn_implementation="eager").eval()
+        tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
+    else:
+        model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
+                                                     optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
+                                                     quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
+                                                     use_cache=True, attn_implementation="eager").eval()
+        tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
+    end = time.perf_counter()
```

Comment:
```python
if repo_id in CHATGLM_IDS:
     model = AutoModel.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True,
                                      optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]),
                                      quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                      torch_dtype=torch.float16, attn_implementation="eager").eval()
    tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
elif repo_id in LLAMA_IDS:
    model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
                                                 optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
                                                 quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                                 use_cache=True, attn_implementation="eager").eval()
    tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
else:
    model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
                                                 optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
                                                 quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                                 use_cache=True, attn_implementation="eager").eval()
    tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
```
->

```python
model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
                                             optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
                                             quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                             use_cache=True, attn_implementation="eager").eval()
tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
```

Reply:
Also update this for `transformers_int4_npu_win`

Reply:
Maybe also update this for `transformers_int4_npu_win`?

Reply:
Please update this in next PR. I want to merge this PR first for test.